RemoteDriverConfig.wiki

```diff
@@ -25,5 +25,11 @@ Once you have selected the browser in the Capability list box, navigate to the a
 
 For more information on the options, please refer to [[DirectDriverConfig.wiki#Options Tab|Options Tab]] in the Direct Driver wiki page.
 
+== Capabilties Management Tab ==
```

Comment:
As this new tab is added to all configurations (and not only to the RemoteDriver, I would recommend to update DirectDriverConfig.wiki with this new feature. Then in RemoteDriverConfig.wiki, just point the user to DirectDriverConfig.wiki as it is currently done for the other tabs.
Also add your following comments as examples:
Use case1: W3C allows extension capabilities, used by vendors: https://www.w3.org/TR/webdriver1/#dfn-extension-capability
Use case2: Some webdriver requests require authentication via capabilities parameter. This enables that.

Reply:
Done.  Please re-review.